Exiled.CustomItems/API/Extensions.cs

```diff
@@ -41,6 +41,31 @@ public static class Extensions
             SpawnLocation.InsideHid,
         };
 
+        /// <summary>
+        /// Resets the player's inventory to the provided list of items and/or customitems names, clearing any items it already possess.
+        /// </summary>
+        /// <param name="player">The player to which items will be given.</param>
+        /// <param name="newItems">The new items that have to be added to the inventory.</param>
+        public static void ResetInventory(this Player player, List<string> newItems)
```

Comment:
You should add the `bool displayMessage = true` parameter and use it below, when you call `CustomItem.TryGive(...)`.

Reply:
In my opinion `bool displayMessage` should be set to `false` by default, because If player receives 8 custom items they will get 8 hints.

Reply:
Yeah, it's fine.